test/test_helper.rb

```diff
@@ -99,6 +102,7 @@ class ActiveSupport::TestCase
 
   teardown do
     Rails.configuration.launch_darkly_client.close
+    DatabaseCleaner.clean
```

Comment:
We could remove this and just clear at the start of each run. This would solve the main problem I see.